op-node/rollup/sync/config.go

```diff
@@ -0,0 +1,8 @@
+package sync
+
+type Config struct {
+	// EngineP2PEnabled is true when the EngineQueue can trigger execution engine P2P sync.
+	EngineP2PEnabled bool `json:"engine_p2p_enabled"`
+	// SkipSanityCheck is true when the EngineQueue does not do sanity check on pipeline reset.
+	SkipSanityCheck bool `json:"skip_sanity_check"`
```

Comment:
Apply lower camel case for consistency.

```suggestion
	// EngineP2PEnabled is true when the EngineQueue can trigger execution engine P2P sync.
	EngineP2PEnabled bool `json:"engineP2PEnabled"`
	// SkipSanityCheck is true when the EngineQueue does not do sanity check on pipeline reset.
	SkipSanityCheck bool `json:"skipSanityCheck"`
```